kong/pdk/log.lua

```diff
@@ -690,6 +690,33 @@ do
     return root
   end
 
+  local prepare_serialize = function(ongx, okong)
```

Comment:
follow the other function definitions, should be:
```
local function prepare_serialize(ongx, okong)
```

Reply:
should we pass `ongx.ctx` `ongx.var` in function's entrance?

Reply:
> local function prepare_serialize(ongx, okong)

great idea